docs/application/web/guides/connectivity/bluetooth.md

```diff
@@ -154,29 +281,31 @@ To use the Bluetooth functionality of the device, you must switch the Bluetooth
 
 6. Set a friendly name for the device using the `setName()` method.
 
-   The name helps to recognize the device in a list of [retrieved devices](#discovering-bluetooth-devices).
+   The name helps to recognize the device in a list of [retrieved devices](#discover-bluetooth-devices).
```

Comment:
Could we combine this sentence with the previous sentence:

6. Set a friendly name for the device using the `setName()` method. The name helps to recognize the device in a list of [retrieved devices](#discover-bluetooth-devices):

Reply:
Done